File: modulo1/segmento3/apunte3/ejercicio2.py
num = int(input("Ingrese un dia 1-Lunes,2-Martes,3-Miercoles,4-Jueves,5-Viernes,6-Sabado,7-Domingo: ") )

if( num == 1 ):
        opcion = "lunes"
elif (num == 2):
        opcion = "martes"
elif (num == 3):
        opcion = "miercoles"
elif (num == 4):
        opcion = "jueves"
elif (num == 5):
        opcion = "viernes"
elif (num == 6):
        opcion = "sabado"
elif (num == 7):
        opcion = "domingo"
else :
        opcion = "opcion incorrecta"
        num = 0

if (num == 0):
    print(opcion)
else:
    print("El dia elegido es "+  opcion)

File: modulo1/segmento3/apunte3IfElse/Ejercicio2.java
import java.util.Scanner;

public class Ejercicio2{
	public static void main(String args[]){
		Scanner teclado = new Scanner(System.in);

		System.out.print("Ingrese un dia (1-Lunes,2-Martes,3-Miercoles, 4-Jueves, 5-Viernes, 6-Sabado, 7-Viernes): ");
		Integer dia = Integer.parseInt(teclado.nextLine());
		String respuesta;

		if(dia== 1){
			respuesta="Lunes";
		}else if(dia==2){
			respuesta="Martes";
		}else if(dia==3){
			respuesta="Miercoles";
		}else if(dia==4){
			respuesta="Jueves";
		}else if(dia==5){
			respuesta="Viernes";
		}else if(dia==6){
			respuesta="Sabado";
		}else if(dia==7){
			respuesta="Domingo";
		}else{
			respuesta="Error";
		}

		System.out.println("El dia elegido es " + respuesta);

	}
}
